Doc/whatsnew/3.13.rst

```diff
@@ -102,6 +102,11 @@ Deprecated
 Removed
 =======
 
+* Remove the ``sndhdr`` module, deprecated in Python 3.11: use the projects
+  `filetype <https://pypi.org/project/filetype/>`_,
+  `puremagic <https://pypi.org/project/puremagic/>`_,
+  or `python-magic <https://pypi.org/project/python-magic/>`_ instead.
+  (Contributed by Victor Stinner in :gh:`104773`.)
```

Comment:
Good idea to list the PyPI alternatives 👍 

Let's also add a reference to `` `:pep:`594` `` for people searching for it.